app/api_alpha/views.py

```diff
@@ -555,7 +557,72 @@ def get_diff(request):
         return response
 
 
-class ContributionsViewSet(viewsets.ModelViewSet):
+class ContributionsViewSet(mixins.CreateModelMixin, viewsets.GenericViewSet):
     queryset = Contribution.objects.all()
-    http_method_names = ["post"]
     serializer_class = ContributionSerializer
+
+    def create(self, request, *args, **kwargs):
+        serializer = ContributionSerializer(data=request.data)
+
+        if serializer.is_valid():
+            serializer.save()
+            if request.query_params.get("classement") == "true" and request.data.get(
```

Comment:
On peut rester en anglais
```suggestion
            if request.query_params.get("ranking") == "true" and request.data.get(
```

Reply:
ok